cmd/xl-v1-object.go

```diff
@@ -113,7 +112,7 @@ func (xl xlObjects) CopyObject(srcBucket, srcObject, dstBucket, dstObject string
 		pipeWriter.Close() // Close writer explicitly signalling we wrote all data.
 	}()
 
-	objInfo, err := xl.PutObject(dstBucket, dstObject, NewHashReader(pipeReader, length, metadata["etag"], ""), metadata)
+	objInfo, err := xl.PutObject(dstBucket, dstObject, NewHashReader(pipeReader, length, "", ""), metadata)
```

Comment:
why etag is not passed?

Reply:
Its not needed and incorrect multipart Object copied to new location would have a wrong ETag and fail.